src/status_im/ui/screens/ens/views.cljs

```diff
@@ -0,0 +1,453 @@
+                             :action-fn     #(re-frame/dispatch [:navigate-to :ens-register])
+                             :icon          :main-icons/delete
+                             :icon-color    colors/gray
+                             :active?       false
+                             :hide-chevron? true}]]]]]))
+
+;; Terms
+
+(defn- term-point [content]
+  [react/view {:style {:flex 1 :margin-top 24 :margin-horizontal 16 :flex-direction :row}}
+   [react/view {:style {:width 16 :margin-top 8}}
+    [react/view {:style {:background-color colors/gray :width 4 :height 4 :border-radius 25}}]]
+   [react/text {:style {:flex 1 :font-size 15}}
+    content]])
+
+(defn- etherscan-url [address]
+  (str "https://etherscan.io/address/" address))
+
+(views/defview terms []
+  (views/letsubs [{:keys [contract]} [:get-screen-params :ens-terms]]
+    [react/scroll-view {:style {:flex 1}}
+     [toolbar/simple-toolbar
+      (i18n/label :t/ens-terms-registration)]
+     [react/view {:style {:height 136 :background-color colors/gray-lighter :justify-content :center :align-items :center}}
+      [react/text {:style {:text-align-vertical :center :text-align :center
+                           :typography :header :letter-spacing -0.275}}
+       (i18n/label :t/ens-terms-header)]]
+     [react/view
+      [term-point
+       (i18n/label :t/ens-terms-point-1)]
+      [term-point
+       (i18n/label :t/ens-terms-point-2)]
+      [term-point
+       (i18n/label :t/ens-terms-point-3)]
+      [term-point
+       (i18n/label :t/ens-terms-point-4)]
+      [term-point
+       (i18n/label :t/ens-terms-point-5)]
+      [term-point
+       (i18n/label :t/ens-terms-point-6)]
+      [term-point
+       (i18n/label :t/ens-terms-point-7)]]
+     [react/view
+      [react/text {:style {:font-size 15 :margin-top 24 :margin-horizontal 16}}
+       (i18n/label :t/ens-terms-point-8)]
+      [term-point
+       (i18n/label :t/ens-terms-point-9 {:address contract})]
+      [react/view {:style {:align-items :center :margin-top 16}}
+       [link {:on-press #(.openURL react/linking (etherscan-url contract))}
+        (i18n/label :t/etherscan-lookup)]]
+      [term-point
+       (i18n/label :t/ens-terms-point-10)]
+      [react/view {:style {:align-items :center :margin-top 16 :margin-bottom 24}}
+       [link {:on-press #(.openURL react/linking (etherscan-url "0x314159265dd8dbb310642f98f50c066173c1259b"))}
+        (i18n/label :t/etherscan-lookup)]]]]))
+
+;; Registration
+
+(defn- valid-domain? [state]
+  (#{:registrable :owned :connected} state))
+
+(defn- final-state? [state]
+  (#{:saved :registered :registration-failed} state))
+
+(defn- main-icon [state]
+  (cond
+    (valid-domain? state)    :main-icons/check
+    (= state :unregistrable) :main-icons/cancel
+    :else                    :main-icons/username))
+
+(defn- icon-wrapper [color icon]
+  [react/view {:style {:margin-right 10 :width 32 :height 32 :border-radius 25 :align-items :center :justify-content :center
```

Comment:
could be defined as a style separately or at least not in 1 line so stayl below char limit

Reply:
I believe @flexsurfer is in favor of having inline styles

Reply:
I am too but just talking about the layout  here

Reply:
we still don't have any agreement on this, imo it's about balance if it's too noisy it can be moved to styles for sure